### DISCUSSION TITLE
Import Error when accessed from iPad Edge browser

### DISCUSSION THREAD

**Discussion by LesWeist** [TheLastBen/fast-stable-diffusion#285] [Q&A] [answered]
I get this error message at the last cell “Start Stable diffusion.” What does it mean and what should I do? I am using an iPad. Are iPads not cut out for Google Colabs?

Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 14, in <module>
    import modules.extras
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/extras.py", line 13, in <module>
    from modules.ui import plaintext_to_html
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/ui.py", line 45, in <module>
    from modules.generation_parameters_copypaste import image_from_url_text
ImportError: cannot import name 'image_from_url_text' from 'modules.generation_parameters_copypaste' (/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/generation_parameters_copypaste.py)

**Comment by TheLastBen** [comment, marked as answer]
try checking the box "update_repo" and run it, if it persists
Open an issue here https://github.com/AUTOMATIC1111/stable-diffusion-webui/ hopefully they'll fix it

**Comment by LesWeist** [comment]
Oh, thanks. That worked. I didn’t see the “Update repo” checkbox on the 2nd cell. Thank you.

**Reply by TheLastBen** [reply]
Great !
